packages/surveys/src/styles/global.css

```diff
@@ -82,3 +82,12 @@ p.fb-editor-paragraph {
   --fb-close-btn-color: var(--slate-500);
   --fb-close-btn-color-hover: var(--slate-700);
 }
+
+@keyframes ShrinkWidthToZero {
```

Comment:
As we currently don't have a naming convention for keyframes I would either prefer lower camel case from javascript `shrinkWidthToZero` or `shrink-width-to-zero` as the other css classes.

Reply:
Changed it to `shrink-width-to-zero` , so that it aligns with other css classes as well 😊